xml/System/Environment+SpecialFolder.xml

```diff
@@ -575,7 +575,7 @@
       </ReturnValue>
       <MemberValue>22</MemberValue>
       <Docs>
-        <summary>The file system directory that contains the programs and folders that appear on the **Start** menu for all users. This special folder is valid only for Windows NT systems. Added in the .NET Framework 4.</summary>
+        <summary>The file system directory that contains the programs and folders that appear on the **Start** menu for all users. This special folder is valid only for Windows NT systems.</summary>
```

Comment:
We seem to [still support this](https://github.com/dotnet/runtime/search?q=extension%3Acs+CommonStartMenu) and the two enums below. I suspect this sentence was added back when Windows NT was the latest OS, but no one updated the sentence to reflect the subsequent OS that supported this. 

```suggestion
        <summary>The file system directory that contains the programs and folders that appear on the **Start** menu for all users.</summary>
```